Makefile.build

```diff
@@ -20,6 +20,20 @@ push-buildenv: build-buildenv
 	@gcloud $(GCLOUD_QUIET) auth configure-docker
 	@docker push $(BUILDENV_IMAGE)
 
+GCB_DOCKER_GCLOUD_VERSION := v20220617-174ad91c3a
+GCP_DOCKER_GCLOUD_TAG := gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:$(GCB_DOCKER_GCLOUD_VERSION)
```

Comment:
why's one of these := and the other ?=

Reply:
The intention is that the variables for the base images are not configurable at runtime, but the destination tag of the built image can be set at runtime